### DISCUSSION TITLE
Ability to move entities

### DISCUSSION THREAD

**Discussion by ihategravel2** [Mojang/minecraft-editor#18] [Feedback]
Let's say you are filling in an area with a huge amount of dirt, and, oh no! My precious pig is right in the middle of where I'm going to put all this dirt. So, you have to go into Crosshair Mode and manually move the pig. Now, imagine if you could just click on your prized pig and move it out of the way. Sounds great, right?

That is basically what I'm suggesting.

**Comment by G01dY05h1** [comment]
(I love the way this is written)

**Comment by minecraftchloe** [comment]
Brilliant - yes, we have planned the ability to select entities in the same way you can select blocks! 

+1 this is written beautifully <3
